pkg/cmd/pulumi/stack_output.go

```diff
@@ -97,22 +104,14 @@ func (cmd *stackOutputCmd) Run(ctx context.Context, args []string) error {
 		name := args[0]
 		v, has := outputs[name]
 		if has {
-			if cmd.jsonOut {
-				if err := printJSON(v); err != nil {
-					return err
-				}
-			} else {
-				fmt.Printf("%v\n", stringifyOutput(v))
+			if err := outw.WriteOne(v); err != nil {
+				return err
 			}
 		} else {
 			return fmt.Errorf("current stack does not have output property '%v'", name)
 		}
-	} else if cmd.jsonOut {
-		if err := printJSON(outputs); err != nil {
-			return err
-		}
-	} else {
-		printStackOutputs(outputs)
+	} else if err := outw.WriteMany(outputs); err != nil {
```

Comment:
I'd split this just to make it clear this is a plain "else" of args =0
```suggestion
	} else { 
	  if err := outw.WriteMany(outputs); err != nil {
```

Reply:
Ack. I flip-flopped on this when writing it, and your comment confirms my suspicion that the other way was more obvious.